
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:4030462) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_3',
  containerIp: '10.0.3.153',
  mitmPort: 1002,
  autoAccessFixed: 1
}
2022-10-23 17:16:43.045 - [Info] MITM Version: 2
2022-10-23 17:16:43.047 - [Info] Auto Access Enabled: true
2022-10-23 17:16:43.048 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-23 17:16:43.055 - [Info] SSH man-in-the-middle server for 10.0.3.153 listening on 10.0.3.1:1002
2022-10-23 17:16:43.380 - [Debug] [Connection] Attacker connected: 129.154.55.234 | Client Identification: SSH-2.0-Go
2022-10-23 17:16:43.645 - [Debug] [Connection] Attacker connected: 59.1.226.211 | Client Identification: SSH-2.0-Go
2022-10-23 17:16:44.202 - [Debug] [Connection] Attacker connected: 31.165.5.151 | Client Identification: SSH-2.0-Go
2022-10-23 17:16:44.591 - [Debug] [Auth] Attacker 129.154.55.234 trying to authenticate with 'none'
2022-10-23 17:16:44.592 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 17:16:44.858 - [Debug] [Connection] Attacker connected: 120.236.78.194 | Client Identification: SSH-2.0-Go
2022-10-23 17:16:44.983 - [Debug] [Auth] Attacker 129.154.55.234 trying to authenticate with 'publickey'
2022-10-23 17:16:44.984 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-23 17:16:44.985 - [Debug] [Auth] Attacker: 129.154.55.234 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 17:16:45.043 - [Debug] [Auth] Attacker 31.165.5.151 trying to authenticate with 'none'
2022-10-23 17:16:45.043 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 17:16:45.344 - [Debug] [Auth] Attacker 31.165.5.151 trying to authenticate with 'publickey'
2022-10-23 17:16:45.346 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-23 17:16:45.347 - [Debug] [Auth] Attacker: 31.165.5.151 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 17:16:45.374 - [Debug] [Connection] Attacker closed the connection
2022-10-23 17:16:45.567 - [Debug] [Connection] Attacker connected: 129.154.55.234 | Client Identification: SSH-2.0-Go
2022-10-23 17:16:45.617 - [Debug] [Connection] Attacker connected: 91.125.119.231 | Client Identification: SSH-2.0-Go
2022-10-23 17:16:45.648 - [Debug] [Connection] Attacker closed the connection
2022-10-23 17:16:45.697 - [Debug] [Connection] Attacker connected: 59.1.226.211 | Client Identification: SSH-2.0-Go
2022-10-23 17:16:45.779 - [Debug] [Connection] Attacker connected: 31.165.5.151 | Client Identification: SSH-2.0-Go
